homeassistant/config.py

```diff
@@ -69,6 +69,7 @@
 GROUP_CONFIG_PATH = "groups.yaml"
 AUTOMATION_CONFIG_PATH = "automations.yaml"
 SCRIPT_CONFIG_PATH = "scripts.yaml"
+SCENE_CONFIG_PATH = "scenes.yaml"
```

Comment:
Can you make sure that we also create this file when we create a new config dir ? See where `SCRIPT_CONFIG_PATH` is used below.